torchvision/prototype/transforms/_utils.py

```diff
@@ -68,5 +68,6 @@ def has_all(sample: Any, *types_or_checks: Union[Type, Callable[[Any], bool]]) -
 # TODO: Given that this is not related to pytree / the Transform object, we should probably move it to somewhere else.
 #  One possibility is `functional._utils` so both the functionals and the transforms have proper access to it. We could
 #  also move it `features` since it literally checks for the _Feature type.
+# datumbox: let's do the move in `features` module.
```

Comment:
cc @pmeier

Reply:
I would leave this last @pmeier. Else we are going to get a billion conflicts with so many PRs open 💥

Reply:
See https://github.com/pytorch/vision/pull/6492/commits/7043b6ee3e3b1f6541371a4f2442cfc1fd664e4a

Reply:
The commit quoted by @pmeier was reverted. This is still pending and should be done last.

Reply:
#6507